packages/grid/x-data-grid-premium/src/DataGridPremium/DataGridPremium.tsx

```diff
@@ -162,6 +168,10 @@ DataGridPremiumRaw.propTypes = {
    * @deprecated Use the `slotProps` prop instead.
    */
   componentsProps: PropTypes.object,
+  /**
+   * Data attributes for the grid root element.
+   */
+  dataProps: PropTypes.object,
```

Comment:
These are here only because I needed to group them in a single field, not because I want to expose this as a public API. IIRC I should mark these as `@ignore - do not document`, right?